Overview.bs

```diff
@@ -1440,40 +1450,47 @@ Some example inputs and the corresponding expansions:
   <tr>
     <td>//foo.bar/{id}</td>
     <td>123</td>
-    <td>//foo.bar/7B</td>
+    <td>//foo.bar/FC</td>
   </tr>
   <tr>
     <td>//foo.bar{/d1,d2,id}</td>
-    <td>123</td>
-    <td>//foo.bar/B/7/7B</td>
+    <td>478</td>
+    <td>//foo.bar/0/F/07F0</td>
   </tr>
   <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
     <td>123</td>
-    <td>//foo.bar/B/7/0/7B</td>
+    <td>//foo.bar/C/F/_/FC</td>
   </tr>
-n    <tr>
+  <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
     <td>baz</td>
-    <td>//foo.bar/z/a/b/baz</td>
-  </tr>
+    <td>//foo.bar/K/N/G/C9GNK</td>
   </tr>
-    <tr>
+  <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
-    <td>az</td>
-    <td>//foo.bar/z/a/0/az</td>
+    <td>z</td>
+    <td>//foo.bar/8/F/_/F8</td>
   </tr>
-  </tr>
-    <tr>
+  <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
     <td>àbc</td>
-    <td>//foo.bar/c/b/%C3%A0/%C3%A0bc</td>
+    <td>//foo.bar/0/0/4/OEG64OO</td>
+  </tr>
+  <tr>
+    <td>//foo.bar{id64}</td>
```

Comment:
Missing a slash in the template (use bar/{id64} or bar{/id64}), same for the example below.